### PR TITLE
Return variances only in VCV transformations

### LINKED ISSUE #75: Modify vcv_cart2local and vcv_local2cart to return a 3x1 matrix when necessary
vcv_cart2local and vcv_local2cart were modified to accept a 3x1 matrix of variances as well as the full 3x3 VCV. However, the code only returns the 3x3 matrix

### PR DESCRIPTION
**Pull request by harry093**
Fix vcv_cart2local and vcv_local2cart to only return a column vector of variances when that was what was supplied. Fixes #75

### CODE REVIEW

**Comment by harry093** [comment]
@BatchelorJ when you get a chance can you have a look at this PR

File: geodepy/statistics.py [hunk 1]
```diff
@@ -26,19 +27,24 @@ def vcv_cart2local(vcv_cart, lat, lon):
...
+            vcv_cart = np.array([[vcv_cart[0, 0], 0.0, 0.0],
+                                 [0.0, vcv_cart[1, 0], 0.0],
+                                 [0.0, 0.0, vcv_cart[2, 0]]])
         elif vcv_cart.shape[1] == 3:
             pass
         else:
             sys.exit('Matrix must be either 3x1 or 3x3')
     else:
-         sys.exit('Matrix must be either 3x1 or 3x3')
+        sys.exit('Matrix must be either 3x1 or 3x3')
```
[the remark this answers is not in the document]
Comment (harry093) [reply geodepy/statistics.py]: No, old habits. I'll fix it

[the remark this answers is not in the document]
Comment (harry093) [reply geodepy/statistics.py]: @BatchelorJ, I finally sorted this

[the remark this answers is not in the document]
Comment (harry093) [reply geodepy/statistics.py]: I think a couple of tests of this are failing. I'll sort that and the doc strings and the get you to review

**Comment by BatchelorJ** [comment]
Ok go ahead and merge this in. I'm not too sure why it's failing some Travis builds, but we'll have to see after merging and resolve any outstanding issues

**Pull request merged**